### PR TITLE
add clipboard support and featureMenu to advanced example

### PR DESCRIPTION
**Pull request by georgios-uber**

### CODE REVIEW

File: examples/advanced/example.js [hunk 1]
```diff
@@ -271,6 +262,60 @@ export default class Example extends Component<
...
+    try {
+      testFeatures = JSON.parse(json);
+      if (Array.isArray(testFeatures)) {
+        testFeatures = {
+          type: 'FeatureCollection',
+          features: testFeatures
+        };
+      }
+      // eslint-disable-next-line
+      console.log('Loaded JSON:', testFeatures);
```
Comment (xintongxia) [inline examples/advanced/example.js]: `eslint-disable-next-line` is intended?

File: examples/advanced/example.js [hunk 2]
```diff
@@ -579,6 +642,37 @@ export default class Example extends Component<
...
+    if (action === 'delete') {
+      const features = [...testFeatures.features];
+      features.splice(index, 1);
+      testFeatures = Object.assign({}, testFeatures, {
+        features
+      });
+    } else if (action === 'split') {
+      // TODO
+    } else if (action === 'info') {
+      // eslint-disable-next-line
```
Comment (xintongxia) [inline examples/advanced/example.js]: `eslint-disable-next-line` is intended?

Comment (georgios-uber) [reply examples/advanced/example.js]: Yes.

File: examples/advanced/example.js (hunk 1 above)
Comment (georgios-uber) [reply examples/advanced/example.js]: Yes. We need a better logging #136 .

**Pull request merged**